process.c

```diff
@@ -1344,6 +1344,7 @@ proc_wait(int argc, VALUE *argv)
  *        Process.setpgid(0, Process.pid)
  *        child1_pgid = Process.getpgid(Process.pid)
  *        puts "Child 1 process group ID is #{child1_pgid} (different from parent's)."
+ *        sleep 3 # To force child 1 to exit later than child 0 exit.
```

Comment:
While this `sleep` does make it very likely that child 1 exits later than child 0, it does not guarantee it. We should instead change the example to not assume that the first `Process.wait` returns child 0 and the second returns child 1.

Reply:
Fixed.